spec/poros/save_request/skip_checker_spec.rb

```diff
@@ -0,0 +1,75 @@
+RSpec.describe SaveRequest::SkipChecker do
```

Comment:
One of the other nice things about `app/poros/` is that they tend to be relatively easy to test, which I think is reflected in the pretty straightforward tests below.